dask/array/slicing.py

```diff
@@ -584,17 +584,21 @@ def take(outname, inname, chunks, index, itemsize, axis=0):
      ('y', 2): (getitem, ('x', 2), ([7],))}
 
     When any indexed blocks would otherwise grow larger than
-    dask.config.array.chunk-size, we split them.
+    dask.config.array.chunk-size, we might split them,
+    depending on the value of ``dask.config.slicing.split-large-chunks``.
 
-    >>> chunks, dsk = take('y', 'x', [(1, 1, 1), (1000, 1000), (1000, 1000)],
-    ...                    [0] + [1] * 6 + [2], axis=0, itemsize=8)
+    >>> import dask
+    >>> with dask.config.set(**{"array.slicing.split-large-chunks": True}):
```

Comment:
```suggestion
    >>> with dask.config.set({"array.slicing.split-large-chunks": True}):
```

I think that `dask.config.set` is smart enough to handle being given a dict as a positional arg